### ISSUE TITLE
Allow passing runtime tab parameters to _CreateNewTabFromSettings

### ISSUE THREAD

**Issue by Don-Vito** [microsoft/terminal#9817]
# Description of the new feature/enhancement
Followup for https://github.com/microsoft/terminal/pull/9813 that manually sets the runtime tab text on the duplicated tab.
Instead of doing this we should allow passing a runtime state to `_CreateNewTabFromSettings.`

**Comment by DHowett** [comment]
Imagine how cool it would be to have a "linked" terminal that followed the original one (if its background changed, the linked terminals would change), etc.

**Comment by DHowett** [comment]
(This is mostly unrelated musing; sorry!)

**Comment by skyline75489** [comment]
Yes, please! My imaginary/future tmux CC implementation would also need this

**Comment by zadjii-msft** [comment]
Hmm... I think this is kinda the same _effect_ as #11913, right?

**Comment by DHowett** [comment]
However, I do think doing this is one way to do that.

**Comment by g0t4** [comment]
@zadjii-msft I think you were right, it seems like #11913 is relevant... I would love to see duplicated tabs maintain runtime settings of the current tab (including zoom/fontsize) in addition to title, etc.

Likewise, duplicate pane should do the same.
